lms/djangoapps/completion/apps.py

```diff
@@ -0,0 +1,15 @@
+"""
+App Configuration for Completion
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from django.apps import AppConfig
+
+
+class CompletionAppConfig(AppConfig):
+    """
+    App Configuration for Completion
+    """
+    name = 'lms.djangoapps.completion'
+    verbose_name = 'Completion'
+    pass
```

Comment:
👍 on the app configuration! Is this `pass` needed?

Reply:
Nope.